src/result-handler.ts

```diff
@@ -76,9 +78,17 @@ export const defaultResultHandler = createResultHandler({
     });
     return createApiResponse(responseSchema);
   },
-  handler: ({ error, input, output, request, response, logger }) => {
+  handler: ({
+    error,
+    input,
+    output,
+    request,
+    response,
+    logger,
+    positiveStatusCode,
```

Comment:
I believe ResultHandler should not receive the code.
It should decide on its own how to respond and the code should have only declarative meaning, consumed by the documentation generator.

Reply:
@RobinTail I'm having the same issue and was also thinking about passing the success status code to the result handler, because how is the result handler supposed to know what the success status code is? I want to return a 201 when a resource has been created, but return a 200 when a resource is updated or fetched. How is the result handler supposed to know which status code to return? It would be nicer to define the success status code in each endpoint's implementation - how should I do this without passing the success code to the result handler?